static/app/components/events/eventStatisticalDetector/breakpointChart.tsx

```diff
@@ -40,7 +40,7 @@ function EventBreakpointChart({event}: EventBreakpointChartProps) {
 
   const {start: beforeDateTime, end: afterDateTime} = useRelativeDateTime({
     anchor: breakpoint,
-    relativeDays: 7,
+    relativeDays: 14,
```

Comment:
Extending it to 14 days before/after to better see any seasonality in the trend.